tests/x11/mac80211_hwsim.pm

```diff
@@ -0,0 +1,219 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: mac80211 hw test
+# Maintainer: Nick Singer <nsinger@suse.de>
+# Tags: poo#20306
+
+use base 'x11test';
+use y2x11test qw(launch_yast2_module_x11);
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+    select_console 'root-console';
+    assert_script_run "modprobe mac80211_hwsim radios=2 |& tee /dev/$serialdev";
+    save_screenshot;
+
+    $self->install_packages;
+    $self->prepare_NM;
+    $self->generate_certs;
+    $self->configure_hostapd;
+    $self->reload_services;
+    type_string "exit\n";
```

Comment:
Don't log out of the root console. You can call `utils::clear_console` if you prefer